project-icp/bee-pollinator-app
# Remove ITSI

This PR removes ITSI features from all layers of the app, including:

- cutting it from environment variables set in deployment scripts
- removing ITSI login code from the Django backend
- removing ITSI features from the frontend, including 'activityMode', some ITSI-specific project redirects and save features, and associated paths and utility functions
- dropping ITSI-specific templates and ITSI-related conditionals from generic templates
- removing Shutterbug

<img width="1251" alt="screen shot 2016-10-17 at 11 59 38 am" src="https://cloud.githubusercontent.com/assets/4165523/19445288/a5604cfc-9461-11e6-8c01-d8d56dcec4d3.png">

Note that only commits fb42ffb and after are from this PR; the prior set are from #8 

**Testing**
- get this branch, `vagrant up --provision app`
- visit `localhost:8000` in the browser and open the console
- verify that you don't see an ITSI login button on the login modal
- click "Guest" to login and verify that you don't see any errors in the console

Connects #6
